src/components/Logo.svelte

```diff
@@ -1,6 +1,6 @@
-<svg width="100%" viewBox="0 0 216 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
having the new svg in the file was causing some sizing and positioning issues, so I just copied what we do in the faucet